### PR TITLE
Datasource: users (plural)

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves https://github.com/Twingate/terraform-provider-twingate/issues/102

## Changes
- added users datasource

### CODE REVIEW

**Review by ekampf — APPROVED** [review]

File: docs/data-sources/users.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `users` (Block List) (see [below for nested schema](#nestedblock--users))
+
+<a id="nestedblock--users"></a>
+### Nested Schema for `users`
+
+Read-Only:
```
Comment (alexmensch) [inline docs/data-sources/users.md:30]: `is_admin` should read `Indicates whether the User is an admin`
`role` should read `Indicates the User's role. Either ADMIN, DEVOPS, SUPPORT, or MEMBER.`

**Review by alexmensch — CHANGES_REQUESTED** [review]
Some copy changes to make

File: docs/data-sources/users.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+<a id="nestedatt--users"></a>
+### Nested Schema for `users`
+
+Read-Only:
+
+- `email` (String)
+- `first_name` (String)
+- `id` (String)
+- `is_admin` (Boolean)
```
[the remark this answers is not in the document]
Comment (vmanilo) [reply docs/data-sources/users.md]: I think we agreed to have both, no?

File: docs/data-sources/users.md (hunk 1 above)
Comment (vmanilo) [reply docs/data-sources/users.md:30]: updated

File: docs/data-sources/users.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vmanilo) [reply docs/data-sources/users.md]: updated description

**Review by alexmensch — APPROVED** [review]

**Pull request merged**